packages/SidePanel/src/components/Group/Group.styles.js

```diff
@@ -7,6 +7,7 @@ export const groupCSS = css`
   justify-content: flex-end;
   position: fixed;
   right: 0;
+  visibility: hidden;
```

Comment:
can you add it to the comments on the styles? or just referred with a link to this comment?

Reply:
Sure